packages/components/select/src/option-group.tsx

```diff
@@ -22,6 +22,8 @@ export const OptionGroup = forwardRef<OptionGroupProps, "ul">(
 
     return (
       <ui.li
+        role="group"
+        aria-label={rest["aria-label"] ?? label}
```

Comment:
I think you should adapt your writing style to that of `Slider`.
https://github.com/yamada-ui/yamada-ui/pull/2060

```tsx
{ className, color, h, height, minH, minHeight, children, "aria-label":ariaLabel, ...rest },

...

role="group"
aria-label=ariaLabel ?? label
```